src/app/common/hooks/balance/use-total-balance.tsx

```diff
@@ -0,0 +1,32 @@
+import { baseCurrencyAmountInQuote } from '@app/common/money/calculate-money';
+import { i18nFormatCurrency } from '@app/common/money/format-money';
+import { useBitcoinAssetBalance } from '@app/query/bitcoin/address/address.hooks';
+import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
+import { useCurrentStacksAccountAnchoredBalances } from '@app/query/stacks/balance/balance.hooks';
+import { useCurrentBtcNativeSegwitAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+export function useTotalBalance() {
+  // get market data
+  const btcMarketData = useCryptoCurrencyMarketData('BTC');
+  const stxMarketData = useCryptoCurrencyMarketData('STX');
+
+  // get stx balance
+  const { data: balances } = useCurrentStacksAccountAnchoredBalances();
+
+  // what is the difference between these two?
+  // const { data: stxUnachoredAssetBalance } = useStacksUnanchoredCryptoCurrencyAssetBalance(address);
```

Comment:
in calculations I used anchored balance. is it right?

Reply:
unanchored  = in microblock
anchored = in actual block